container/mock_docker_client.go

```diff
@@ -0,0 +1,89 @@
+// +build ignore
+package container
+
+import (
+	"errors"
+
+	"github.com/fsouza/go-dockerclient"
+)
+
+type MockDockerClient struct {
```

Comment:
shouldn't the file name be `mock_docker_client_test.go` so this code is not included in the binary?